tests-legacy/Unit/Classes/Module/ModuleCoreTest.php

```diff
@@ -88,4 +89,38 @@ public function testDisplayErrorShouldReturnMultipleErrors()
         $crawler = new Crawler($htmlOutput);
         $this->assertCount(3, $crawler->filter('.module_error li'));
     }
+
+    /**
+     * @param int $multistoreCompatibility
+     * @param bool $throwsException
+     *
+     * @dataProvider setMultistoreCompatibilityProvider
+     */
+    public function testSetAndGetMultistoreCompatibility(int $multistoreCompatibility, bool $throwsException): void
+    {
+        $module = new FakeModule();
+        if ($throwsException) {
+            $this->expectException(PrestaShopExceptionCore::class);
+        }
+
+        $module->setMultistoreCompatibility($multistoreCompatibility);
+
+        if (!$throwsException) {
+            $this->assertEquals($multistoreCompatibility, $module->getMultistoreCompatibility());
+        }
```

Comment:
You can replace this by a test that verifies the default value for getMultistoreCompatibility